data/changelog.txt

```diff
@@ -1,3 +1,584 @@
+# 0.G (Gernsback or Gaiman are my suggestions/guess)
+
+## Highlights
+Effect on condition brings a scripting pseudo language to Cataclysm.  Some of its key uses in 0.G have been the introduction of portal storms, NPC faction development over time and random encounters in already explored maps.
+The introduction of hobbies has added a new tab to character creation that allows the player to select one or more hobbies/backgrounds that flesh out what your character was doing prior to the Cataclysm in skills and traits.
+JSONized sidebar allows almost total control of what your sidebar looks like and what information it displays is up to you!
+Skill rust has been turned into a net gain for the player as any rusted xp now turns into an xp multiplier that helps your character learn faster.  As a result it now defaults to on.
+Passively boil water by putting dirty water into a fireproof container then placing the container on top of a fire.  Walk away and after an amount of time dependent on the amount of water to boil it will all be clean water!
+Monster weakpoints! Many monsters now have weakpoints that the player can hit to apply various effects and extra damage to the monster.  As the player dissects more corpses of monsters they can learn weakpoint proficiencies that increase the player's odds of hitting weakpoints.
+Characters now have cardio which will respond to your playstyle.  Go for a run every day and perform hard labor and your stamina will slowly go further and further.  Spend three months reading and performing low intensity crafts and your cardio will suffer.
+A half dozen new in repo mods: Innawoods for the survival from scratch you crave. Backrooms which takes place in the neverending backrooms dimension, Massachusetts mod for people who desire geographic accuracy, Xedra Evolved a science fantasy mod with weird magic and vampirism, Tropicataclysm for your Cataclysm at a warmer latitude and the Megafauna mod which targets an ancient epoch with creatures that outsize our modern equivalents, similar to Dinomod.
+Take over a friendly NPC upon your death, if you've managed to recruit any NPCs to your faction you are given an option to take over one of them in the event of your character's death.
+Appliances and building based powergrids, allow you to set up various tools or even usable arcade machines in your home base.
+Two new Iso tilesets have been added to the game.
+Simple monster aggression differentation. Some monsters now know if another monster made them angry versus instead the PC.  If they get angry at other monsters they may not get angry at the PC.
```

Comment:
```suggestion
Simple monster aggression differentation. Some monsters now know if another monster made them angry instead of the PC.  If they get angry at other monsters they may not get angry at the PC.
```